src/executor.ts

```diff
@@ -610,6 +610,21 @@ function setupExpoDir(dir, platform) {
     if (appJson.expo.icon) {
         appJson.expo.icon = path.join(path.resolve('.'), appJson.expo.icon);
     }
+    // TODO: Temporary native base support
+    // https://github.com/sysgears/spin.js/issues/2#issuecomment-327819766
+    for (let name of Object.keys(builders)) {
+        const builderVal = builders[name];
+        const builder = (typeof builderVal === 'object' && builderVal.constructor !== Array) ?
+        {...builderVal} : {stack: builderVal};
```

Comment:
Is this needed? I think builder is already prepared as need here:
https://github.com/sysgears/spin.js/blob/master/src/configRc.ts#L35